input_parser.star

```diff
@@ -328,25 +338,26 @@ DEFAULT_OP_STACK_ARGS = {
         {
             "participants": [
                 {
+                    # OP Rollup configuration
                     "el_type": "op-geth",
                     "el_image": "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-geth:v1.101500.0-rc.3",
                     "cl_type": "op-node",
                     "cl_image": "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-node:v1.11.0-rc.2",
                     "count": 1,
                 },
             ],
-            # "batcher_params": {
-            #     "image": "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-batcher",
-            # },
-            # "proposer_params": {
-            #     "image": "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-proposer",
-            # },
+            #     "batcher_params": {
+            #         "image": "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-batcher",
+            #     },
+            #     "proposer_params": {
+            #         "image": "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-proposer",
+            #     },
```

Comment:
If we don't need these here, let's delete. If we want to keep references to a config, we can make a yml file in the github folder and document maybe.